src/config.rs

```diff
@@ -166,6 +166,12 @@ impl IndexMut<usize> for ColorList {
     }
 }
 
+#[derive(Debug, Deserialize)]
+pub struct Shell {
+    program: Option<String>,
+    args: Option<Vec<String>>,
```

Comment:
The `Option` is a bit redundant here. The empty `Vec` is equivalent to `None`, and dropping the option would make it a bit easier to work with.

A helper method to return `&[String]` would be nice.